ui/selectors/selectors.js

```diff
@@ -532,6 +532,28 @@ export function getTotalUnapprovedMessagesCount(state) {
   );
 }
 
+export function getTotalUnapprovedSignatureRequestCount(state) {
+  const {
+    unapprovedMsgCount = 0,
```

Comment:
Are these values falsy/not populated in some cases?

Reply:
Actually not sure, I just copied this over of the old code, this is a safe change anyways.